### PR TITLE
Topics: Removed topics starting with N #2

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary

Removed topics starting N:

- [x] national-aeronautics-and-space-administration
- [x] national-oceanic-and-atmospheric-administration
- [x] national-telecommunications-and-information-administration
- [x] niaid
- [x] nih
- [x] nist
- [x] nlm
- [x] noaa
- [x] norad
- [x] north-american-defense-command
- [x] nos
- [x] nps
- [x] nrel
- [x] nsf
- [x] nuclear-regulatory-commission
- [x] nws

### Removal Steps
- [x] removed topics starting with N in `/content/topics/[topic]`
- [x] removed topic tag in the `topics` front matter field
- [x] archived pages on wayback machine
- [ ] replace `agile` placeholder topic for affected files

### Replace Placeholder Topic

- [ ] content/news/2022/01/2022-01-11-celebrating-digital-gov-communities-2021-in-review.md

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

### CODE REVIEW

**Review by mejiaj — APPROVED** [review]
LGTM.

Tested for topics with this regex:

```
^\s*-\s* (national-aeronautics-and-space-administration | national-oceanic-and-atmospheric-administration | national-telecommunications-and-information-administration | niaid | nih | nist | nlm | noaa | norad | north-american-defense-command | nos | nps | nrel | nsf | nuclear-regulatory-commission | nws)$
```

Found no references to `topics/{{ TOPIC }}` in shortcodes.

File: content/news/2014/06/2014-06-13-challenge-and-prize-practitioners-gather-at-gsa.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ topics:
   - nasa
   - National Eye Institute
   - national-science-foundation
-  - nsf
   - recaps
```
Comment (bonnieAcameron) [inline content/news/2014/06/2014-06-13-challenge-and-prize-practitioners-gather-at-gsa.md:19]:
```suggestion
  - challenge-and-prize-competition
```

File: content/news/2014/10/2014-10-06-pillbox-google-hangout-for-developers.md [hunk 2]
```diff
@@ -12,8 +12,6 @@ topics:
   - HHS
   - national-institutes-of-health
   - National Library of Medicine
-  - nih
-  - NLM
   - open-source
```
Comment (bonnieAcameron) [inline content/news/2014/10/2014-10-06-pillbox-google-hangout-for-developers.md:15]:
```suggestion
  - software-development
```

File: content/news/2014/12/2014-12-08-2nd-u-s-public-participation-playbook-draft-responds-to-public-contributions.md [hunk 3]
```diff
@@ -27,9 +27,6 @@ topics:
   - National Defense University
   - national-institutes-of-health
   - National Library of Medicine
-  - nih
-  - NLM
-  - NPS
   - ODNI
```
Comment (bonnieAcameron) [inline content/news/2014/12/2014-12-08-2nd-u-s-public-participation-playbook-draft-responds-to-public-contributions.md:30]:
```suggestion
  - ODNI
  - open-government
```

File: content/news/2014/12/2014-12-08-crowdsourcing-month-an-overview.md [hunk 4]
```diff
@@ -31,7 +31,6 @@ topics:
   - nasa
   - national-archives-and-records-administration
   - national-science-foundation
-  - nsf
   - us-agency-for-international-development
```
Comment (bonnieAcameron) [inline content/news/2014/12/2014-12-08-crowdsourcing-month-an-overview.md:34]:
```suggestion
  - open-government
```

File: content/news/2014/12/2014-12-09-can-you-crowdsource-your-user-experience-research.md [hunk 5]
```diff
@@ -14,7 +14,6 @@ topics:
   - crowdsourcing
   - data-gov
   - Federal CrowdSource Mobile Testing Program
-  - NREL
   - usability
```
Comment (bonnieAcameron) [inline content/news/2014/12/2014-12-09-can-you-crowdsource-your-user-experience-research.md:17]:
```suggestion
  - open-government
```

File: content/news/2014/12/2014-12-18-10-years-of-digital-government-a-retrospective.md [hunk 6]
```diff
@@ -19,8 +19,6 @@ topics:
   - NARA
   - nasa
   - national-archives-and-records-administration
-  - national-oceanic-and-atmospheric-administration
-  - NOAA
   - Office of Management and Budget
```
Comment (bonnieAcameron) [inline content/news/2014/12/2014-12-18-10-years-of-digital-government-a-retrospective.md:22]:
```suggestion
  - open-government
```

File: content/news/2014/12/2014-12-22-how-six-agencies-are-crowdsourcing-with-mobile-apps.md [hunk 7]
```diff
@@ -13,8 +13,6 @@ topics:
   - federal-emergency-management-agency
   - fema
   - nasa
-  - national-oceanic-and-atmospheric-administration
-  - NOAA
   - transportation-security-administration
```
Comment (bonnieAcameron) [inline content/news/2014/12/2014-12-22-how-six-agencies-are-crowdsourcing-with-mobile-apps.md:16]:
```suggestion
  - open-government
  - mobile
```

File: content/news/2014/12/2014-12-23-announcing-the-winners-of-the-first-ever-white-house-3d-printed-ornament-challenge.md [hunk 8]
```diff
@@ -11,7 +11,6 @@ topics:
   - monthly-theme
   - nasa
   - national-institutes-of-health
-  - nih
   - smithsonian-institution
```
Comment (bonnieAcameron) [inline content/news/2014/12/2014-12-23-announcing-the-winners-of-the-first-ever-white-house-3d-printed-ornament-challenge.md:14]:
```suggestion
  - emerging-tech
```

File: content/news/2014/12/2014-12-31-big-in-socialgov-in-2014-services-access-and-participation.md [hunk 9]
```diff
@@ -21,9 +21,6 @@ topics:
   - national-archives-and-records-administration
   - National Highway Traffic Safety Administration
   - National Library of Medicine
-  - national-oceanic-and-atmospheric-administration
-  - NLM
-  - NOAA
   - ODNI
   - peace-corps
```
Comment (bonnieAcameron) [inline content/news/2014/12/2014-12-31-big-in-socialgov-in-2014-services-access-and-participation.md:25]:
```suggestion
  - peace-corps
  - open-government
```

File: content/news/2015/01/2015-01-29-nasa-reaches-new-frontiers-in-3d-printing.md [hunk 10]
```diff
@@ -11,7 +11,6 @@ topics:
   - challenges-and-prize-competitions
   - nasa
   - national-institutes-of-health
-  - nih
   - small-business-innovation-research
```
Comment (bonnieAcameron) [inline content/news/2015/01/2015-01-29-nasa-reaches-new-frontiers-in-3d-printing.md:14]:
```suggestion
  - emerging-tech
```

File: content/news/2015/02/2015-02-11-new-nih-clinical-trials-media-player-now-available.md [hunk 11]
```diff
@@ -10,7 +10,6 @@ topics:
   - cdc
   - centers-for-disease-control-and-prevention
   - national-institutes-of-health
```
Comment (bonnieAcameron) [inline content/news/2015/02/2015-02-11-new-nih-clinical-trials-media-player-now-available.md:12]:
```suggestion
  - emerging-tech
```

File: content/news/2015/08/2015-08-20-a-six-month-update-on-how-weve-been-using-data-and-how-it-benefits-all-americans.md [hunk 12]
```diff
@@ -11,7 +11,6 @@ topics:
   - Commerce
```
Comment (bonnieAcameron) [inline content/news/2015/08/2015-08-20-a-six-month-update-on-how-weve-been-using-data-and-how-it-benefits-all-americans.md:11]:
```suggestion
  - open-data
```

File: content/news/2015/08/2015-08-28-digitalgov-summit-recap-building-privacy-identity-management-in-the-open.md [hunk 13]
```diff
@@ -11,7 +11,6 @@ topics:
   - our-work
   - digitalgov-summit
   - DOT
```
Comment (bonnieAcameron) [inline content/news/2015/08/2015-08-28-digitalgov-summit-recap-building-privacy-identity-management-in-the-open.md:13]:
```suggestion
  - DOT
  - privacy
```

File: content/news/2015/10/2015-10-07-is-your-agency-winning-its-mobile-moments.md [hunk 14]
```diff
@@ -12,8 +12,6 @@ topics:
   - Federal CrowdSource Mobile Testing Program
   - Internal Revenue Service
   - IRS
-  - national-oceanic-and-atmospheric-administration
-  - NOAA
   - transportation-security-administration
```
Comment (bonnieAcameron) [inline content/news/2015/10/2015-10-07-is-your-agency-winning-its-mobile-moments.md:15]:
```suggestion
  - mobile
```

File: content/news/2015/10/2015-10-08-white-house-kicks-off-challenge-gov-anniversary-with-wealth-of-new-prize-competitions.md [hunk 15]
```diff
@@ -13,7 +13,6 @@ topics:
   - DOE
   - epa
   - nasa
-  - NPS
   - OSTP
```
Comment (bonnieAcameron) [inline content/news/2015/10/2015-10-08-white-house-kicks-off-challenge-gov-anniversary-with-wealth-of-new-prize-competitions.md:16]:
```suggestion
  - OSTP
  - challenge-and-prize-competition
```

File: content/news/2015/10/2015-10-26-nihdigital-summit-optimizing-digital-to-reach-patients-scientists-clinicians-and-the-public.md [hunk 16]
```diff
@@ -13,7 +13,6 @@ topics:
   - national-cancer-institute
   - national-institutes-of-health
   - NCI
-  - nih
   - US Department of Agriculture
```
Comment (bonnieAcameron) [inline content/news/2015/10/2015-10-26-nihdigital-summit-optimizing-digital-to-reach-patients-scientists-clinicians-and-the-public.md:16]:
```suggestion
  - open-government
```

File: content/news/2015/12/2015-12-08-challenges-crowdsourcing-a-quick-overview-and-look-ahead.md [hunk 17]
```diff
@@ -20,9 +20,6 @@ topics:
   - FCPCCS
   - national-institute-of-standards-and-technology
   - national-science-foundation
-  - NIST
-  - NPS
-  - nsf
   - OSTP
```
Comment (bonnieAcameron) [inline content/news/2015/12/2015-12-08-challenges-crowdsourcing-a-quick-overview-and-look-ahead.md:23]:
```suggestion
  - OSTP
  - challenge-and-prize-competition
```

File: content/news/2016/08/2016-08-25-how-creating-on-the-fly-digital-content-can-fill-an-immediate-audience-need.md [hunk 18]
```diff
@@ -15,7 +15,6 @@ topics:
   - National Center for Complementary and Integrative Health
   - national-institutes-of-health
   - NCCIH
```
Comment (bonnieAcameron) [inline content/news/2016/08/2016-08-25-how-creating-on-the-fly-digital-content-can-fill-an-immediate-audience-need.md:17]:
```suggestion
  - user-experience
  - content-strategy
```

File: content/news/2016/10/2016-10-03-ideas-on-how-to-keep-your-citizens-informed.md [hunk 19]
```diff
@@ -16,7 +16,6 @@ topics:
   - fda
   - HHS
   - national-institutes-of-health
-  - nih
   - us-food-and-drug-administration
```
Comment (bonnieAcameron) [inline content/news/2016/10/2016-10-03-ideas-on-how-to-keep-your-citizens-informed.md:19]:
```suggestion
  - open-government
```

File: content/news/2016/11/2016-11-16-meeting-patients-where-they-are-liberating-clinical-trials-data-under-the-cancer-moonshot.md [hunk 20]
```diff
@@ -20,7 +20,6 @@ topics:
   - national-cancer-institute
   - national-institutes-of-health
   - NCI
-  - nih
   - PIF
   - Presidential Innovation Fellows
```
Comment (bonnieAcameron) [inline content/news/2016/11/2016-11-16-meeting-patients-where-they-are-liberating-clinical-trials-data-under-the-cancer-moonshot.md:24]:
```suggestion
  - open-data
```

File: content/news/2017/10/2017-10-03-machine-learning-paired-with-skilled-data-scientists-future-datadriven-decision-making.md [hunk 21]
```diff
@@ -15,7 +15,6 @@ topics:
   - defense-advanced-research-projects-agency
   - hhs
   - nasa
```
Comment (bonnieAcameron) [inline content/news/2017/10/2017-10-03-machine-learning-paired-with-skilled-data-scientists-future-datadriven-decision-making.md:17]:
```suggestion
  - nasa
  - open-data
  - analytics
```

**Review by bonnieAcameron — COMMENTED** [review]
I've added topics where I anticipate the remaining topics will all be deleted and not replaced, so that we're not left with 0.

File: content/news/2015/02/2015-02-11-new-nih-clinical-trials-media-player-now-available.md (hunk 11 above)
Comment (ToniBonittoGSA) [reply content/news/2015/02/2015-02-11-new-nih-clinical-trials-media-player-now-available.md:12]: "emerging tech" is stuff like AI, VR/AR, new technologies

File: content/news/2015/10/2015-10-26-nihdigital-summit-optimizing-digital-to-reach-patients-scientists-clinicians-and-the-public.md (hunk 16 above)
Comment (ToniBonittoGSA) [reply content/news/2015/10/2015-10-26-nihdigital-summit-optimizing-digital-to-reach-patients-scientists-clinicians-and-the-public.md:16]: This has product-management (changing to product-and-project-management ?) and social-media as topics; just leave these two
Open gov't is more about making gov't data/info available to the public

**Pull request merged**